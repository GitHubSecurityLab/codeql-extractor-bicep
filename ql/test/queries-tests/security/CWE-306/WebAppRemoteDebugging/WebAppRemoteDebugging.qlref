security/CWE-306/WebAppRemoteDebugging.ql
