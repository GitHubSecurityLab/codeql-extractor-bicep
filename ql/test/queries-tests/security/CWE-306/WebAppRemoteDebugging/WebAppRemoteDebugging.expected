| app.bicep:4:1:12:1 | AppService[webAppBad1] | Azure Web App has remote debugging enabled, which should not be used in production environments |
| app.bicep:15:1:24:1 | AppService[webAppBad2] | Azure Web App has remote debugging enabled, which should not be used in production environments |
